### DISCUSSION TITLE
Highlight preselect item

### DISCUSSION THREAD

**Discussion by kristijanhusak** [hrsh7th/nvim-cmp#1829] [Q&A]
If [preselect](https://github.com/hrsh7th/nvim-cmp/blob/main/doc/cmp.txt#L473-L480) option is enabled (set to `item`), is there a way to add the PmenuSel highlight to the first item that will be automatically selected on `confirm()`?

**Comment by fitrh** [comment]
Add `completeopt` to `completion` config

```lua
preselect = cmp.PreselectMode.Item,
completion = { completeopt = "menu,menuone" },
```

You may also need to adjust `window.completion.winhighlight` config

```lua
window = {
  completion = {
    winhighlight = "Nomal:Pmenu,CursorLine:PmenuSel,Search:None",
  },
},
```
